.changeset/wicked-humans-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": minor
```

Comment:
```suggestion
"@rocket.chat/meteor": patch
```

it needs to be flagged as `patch` if we want to target a patch release.. otherwise it will bump a minor version